AssetsView/Winforms/TextureViewer.cs

```diff
@@ -124,6 +126,31 @@ private void TextureViewer_FormClosed(object sender, FormClosedEventArgs e)
             }
         }
 
+        public void SaveTexture()
+        {
+            var res = MessageBox.Show("Would you like to save the texture as .PNG?", "Texture", MessageBoxButtons.YesNo);
```

Comment:
This dialog is kind of unnecessary since the user already clicked the export button and the extension is shown in the save file dialog.